README.md

```diff
@@ -104,23 +113,68 @@ $ docker-compose ps
+
+## FAQ
+
+### I'm running geth, do I need to run Parity instead?
+Yes - Parity delivers the necessary articulated traces so that TrueBlocks can build its address index. We don't yet support Geth.
+
+### What is my RPC endpoint?
+
+That depends on how you've configured your parity node. some endpoints that we use are http://172.17.0.1:8545 (when running node on linux host), http://docker.for.mac.localhost:8545 (when running node on mac host), and http://my.ethchain.public.dappnode.eth:8545 (when running node on dappnode). http://localhost:8545 is a common guess, but docker has its own networking paradigm; don't be surprised if this RPC endpoint choice doesn't yield results.
+
+Parity's default http RPC port is 8545, but your node could be configured differently.
+
+### Why does it take so long to build the index?
+TrueBlocks has to manually process every single block in the history of the Ethereum chain. Additionally, it has to descend into every transaction trace. Often, traces are deeply layered (traces of traces of traces of ... traces), and this takes time to 1) fetch from your node's RPC and 2) extract addresses.
+
+### Why do we need tracing enabled?
+
+By visiting every bit of the Ethereum data including blocks, transactions, receipts, logs, traces, trace actions, and trace results for every block, TrueBlocks is able to extract every 'appearance' of an address on the blockchain. TrueBlocks can only do this with a tracing node.
+
+### How do you recommend I run a node?
+
+You can run a node with the [DNP_ETHCHAIN](https://github.com/dappnode/DNP_ETHCHAIN) package - set `EXTRA_OPTS=--tracing on` in the dappnode admin panel or setting it in DNP_ETHCHAIN's ethchain.dnp.dappnode.eth.env file (see [Why do we need --tracing enabled?](#why-do-we-need-tracing-enabled))
+
+Otherwise, in order for TrueBlocks to work properly, you need to start parity with at least the following options:
 
 ```
-$ docker run --rm --privileged alpine hwclock -s
+parity --tracing on --jsonrpc-cors all --jsonrpc-hosts all --db-compaction=ssd 
 ```
 
+`--db-compaction=ssd` is optional, but Parity recommends using an SSD drive for storing its data. Enabling `--tracing on` requires a re-sync of your node.
+
+We also recommend (although, this is optional) that you specify an alternative path to store Parity's data (this will also require a re-sync of your node). We suggest an external 4TB SSD.
+
+```
+--db-path=/path/to/larger/hard/drive/      ; optionally add to the command line
+```
+
+Finally, TrueBlocks works equally with both tracing and archive nodes. You may start Parity as an archive node by adding the following option:
```

Comment:
It actually works better with --pruning archive because it can deliver historical balances which is not possible without archive.

Reply:
Do we have a user path for someone who has a pruning node? Like, do we have a section for them in a wiki where they could go to figure out how this distinction could matter?

Would these historical balances data appear in the /export JSON? Or are they accessible only by another tool entirely?

I'll leave this as is here, and follow up in #38.